<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cotação de Todas Moedas</string>
  </property>
  <property name="windowIcon">
   <iconset>
     <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QPushButton" name="consultaButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="conteudoTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
